targets/grandcentral-m4.json

```diff
@@ -0,0 +1,11 @@
+{
+    "inherits": ["atsamd51p20a"],
+    "build-tags": ["grandcentral_m4"],
+    "flash-1200-bps-reset": "true",
+    "flash-method": "msd",
+    "msd-volume-name": "GCM4BOOT",
+    "msd-firmware-name": "firmware.uf2",
+    "openocd-transport": "swd",
```

Comment:
openocd-target is not needed since it is defined in atsamd51p20a.json

Reply:
Thanks, will remove